### DISCUSSION TITLE
運用人工智慧對抗氣候變遷的十種辦法

### DISCUSSION THREAD

**Discussion by wh20502** [1101-GS3073-C/Python-Community-GS3073-C#202] [科技新知]
我覺得科技發展在未來是不可避免的，還有一個時常拿來炒作的就是綠能，剛好找到一篇討論永續發展氣候變遷的文章，裡面有說明十種可能的辦法，第一種的再生能源利用演算法決定分配方式與使用時機我覺得非常實用，未來科技繼續發展，能源肯定是一個主題，若是能夠節能或是更有效的分配能源，對人類發展無疑是有利的。

新聞來源：
https://npost.tw/archives/52639

**Comment by minniewei** [comment]
找到節能減碳的方法刻不容緩，我覺得這篇文章寫得很實用

**Comment by handsomeniko** [comment]
不僅氣候相關，其實在地震預警方面也開始運用到AI了。

**Comment by Shiba010** [comment]
長知識了，原來AI的應用範圍可以這麼廣。

**Comment by lx3211** [comment]
現在使用AI識別瀕臨物種相片比起人力又快上許多...

**Comment by NcuJerryKao** [comment]
若連氣候也能用AI解決
那我想請AI分析 如何對抗宇宙外的力量 或是與之共協
如何解決蜥蜴人的隱憂

感覺有什麼深難的問題 問AI準沒錯
不過應該會如同電影那樣 跳出資料不足 無法評估

**Comment by 32844583** [comment]
簡單來說，AI 可以通過分析大量氣候數據來預測氣候變遷的趨勢，並對未來的氣候變化作出評估。這可以幫助政策制定者做出明智的決策

**Comment by BingRong1217** [comment]
如果可以讓科技解決科技製造出的問題，那也是很不錯的

**Comment by skyling0299** [comment]
利用AI可以幫我們做到更多平常做不到的事

**Comment by limitx0** [comment]
用AI來節能的確是很好的方法

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方
